### DISCUSSION TITLE
OPA-27-Procedure-Currency

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#529] [Q&A] [answered]
Hi Labs,

I have a question about _OPA-27-Procedure-Currency_ field.
If I want to use the _BT-27-Procedure_ field I can set the currency in the _OPA-27-Procedure-Currency_ field.

1.What about if I want to use the _BT-27-Lot_ field? 
Which BT/OPA defines the currency?
Does it use the OPA-27 field too? 

2.The question is tha same about BT-271? 
Which BT/OPA defines the currency? 
Does it use the OPA-27 field too or not? 

Thanks in advance
Anna

PS: SDK 1.7.0

**Comment by AnnaKSZ** [comment]
Another question: 3.Can the currencies of  BT-27-Procedure and BT-27-Lot differ from each other?
Is it a valid case to have EUR in procedure currency field and not EUR in the lot currency field?

Thanks in advance

**Comment by AnnaKSZ** [comment]
4. Another similar question: the question the same as above with BT-118 - OPA-118 and BT-1118.
Which BT/OPA defines the currency of BT-1118?

**Comment by YvesJo** [comment, marked as answer]
the OPA fields were introduced for other reasons than notice edition. To simplify the generation of the XML and get rid of the currencies implicit rule for the currencyID, fields will be defined and associated to the attributes starting SDK 1.9. The fields associated to attributes will be defined in fields.json where their association to a given field will also be specified.

**Reply by AnnaKSZ** [reply]
So there will be more OPA fields in the future. 
Thanks

**Reply by elerik** [reply]
Thats very good news ! :) Thanks
